frontend/src/Components/Markdown/InlineMarkdown.js

```diff
@@ -47,7 +48,11 @@ class InlineMarkdown extends Component {
           markdownBlocks.push(data.substr(endIndex, match.index - endIndex));
         }
 
-        markdownBlocks.push(<code key={`code-${match.index}`}>{match[0].substring(1, match[0].length - 1)}</code>);
+        if (blockClassName) {
+          markdownBlocks.push(<code key={`code-${match.index}`} className={blockClassName}>{match[0].substring(1, match[0].length - 1)}</code>);
+        } else {
+          markdownBlocks.push(<code key={`code-${match.index}`}>{match[0].substring(1, match[0].length - 1)}</code>);
+        }
```

Comment:
```suggestion
        markdownBlocks.push(<code key={`code-${match.index}`} className={blockClassName ?? ''}>{match[0].substring(1, match[0].length - 1)}</code>);
```

I think we could do that, but if that's not working I think this would be a little DRYer.

```
        const codeBlockKey = `code-${match.index}`;
        const codeBlockContent = match[0].substring(1, match[0].length - 1);
        
        if (blockClassName) {
          markdownBlocks.push(<code key={codeBlockKey} className={blockClassName}>{codeBlockContent}</code>);
        } else {
          markdownBlocks.push(<code key={codeBlockKey}>{codeBlockContent}</code>);
        }
```

Reply:
the first suggestion results in <code class> being created, but null instead of '' works